x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -1218,7 +1221,9 @@ response_json using exportId as '2212':
 This behaviour of targeted fixed pattern replacement in the url helps solve various use cases.
 
 NOTE: Fixed patterns must not contain commas in their definition. String replacement patterns are matched by the 
-`replace_with` processor with exact string matching.
+`replace_with` processor with exact string matching. The `value` should not start with a `.` (dot). For example, if we want to
+extract the value from the last_response context object, the `replace_with` clause should look like: `replace_with:'$.xyz_pattern,last_response.body.id'`
+and not `replace_with:'$.xyz_pattern,.last_response.body.id'`.  
```

Comment:
Will be making the improvement so that it looks for 1st word and not 1st element. So this text will be deprecated and removed. It won't matter anymore if dot is present or not and will make things less confusing.